sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/samples/Azure.Messaging.EventHubs.CheckpointStore.Blob.Samples.csproj

```diff
@@ -11,20 +11,6 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(IsTargetingNetStandard)' == 'true'">
-    <PackageReference Include="System.Runtime.Serialization.Primitives" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Azure.Storage.Blobs" />
-    <PackageReference Include="System.Reflection.TypeExtensions" />
-    <PackageReference Include="System.Net.WebSockets.Client" />
-    <PackageReference Include="System.ValueTuple" />
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" />
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" />
-  </ItemGroup>
-
   <ItemGroup>
     <ProjectReference Include="..\src\Azure.Messaging.EventHubs.CheckpointStore.Blob.csproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\Azure.Messaging.EventHubs\samples\Azure.Messaging.EventHubs.Samples.csproj" />
```

Comment:
Just curious, why are we giving EventHubs samples reference(`Azure.Messaging.EventHubs\samples\Azure.Messaging.EventHubs.Samples.csproj`) here?

Reply:
Good point. I don't think we need it.

Reply:
We don't.   Shivangi and I were able to confirm.  Update coming once tests pass.